src/components/select/filterable-select/filterable-select.stories.mdx

```diff
@@ -210,6 +210,59 @@ A custom `Button` Component could be passed as the `listActionButton` value.
   </Story>
 </Preview>
 
+### Asynchronous:
+
+<Preview>
+  <Story name="asynchronous" parameters={{ chromatic: { disable: true }}} > 
+    {() => {
+      let isInitial = useRef(true);
```

Comment:
I think we can simplify this by using `useState` instead of `useRef`? Same in the simple story.

Reply:
State should be used only when something is being passed down through a prop, as modification of it's value will cause re-render. I'll leave it as it is, using state here might be a bad practice.